ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+
```

Comment:
?

Reply:
It is just for the readiness

Reply:
Do you want that i remove the empty line ?